### DISCUSSION TITLE
brew install gcc after sudo apt install build-essentials on debian?

### DISCUSSION THREAD

**Discussion by bc1121** [Homebrew/discussions#1456] [Getting started]
Should I still `brew install gcc` after `sudo apt install build-essentials` on debian?

I am using only current and popular debian distros.
After installing Homebrew, Homebrew recommends `brew install gcc`
I know that debian gcc packages trail brew gcc packages by version #s, but I am inclined to keep my distro package installed and updated by apt to stay in step with the distro build-essential package.

Will brew warn me if it needs a newer version of gcc? Are there any other downsides to keeping my distro gcc as long as I am updating and using well maintained and updated packages?

**Comment by SMillerDev** [comment]
Brew will need it's own `gcc` no matter what you installed from your main OS.

**Reply by bc1121** [reply]
Thanks for your reply! :)
